WinboxExploit.py

```diff
@@ -58,5 +66,9 @@
      d = bytearray(s.recv(1024))
 
      #Get results
-     print(ip)
-     dump(d[55:])
+     print("Connected to " + ip + ":" + str(port))
+     if len(d[55:]) > 25:
```

Comment:
Not 100% sure that `if len(d[55:]) > 25:` is a valid check to see if it worked or not, but it worked in my test setup.

Reply:
It's okay. We'll test it out. >25 seems like a good idea

Reply:
It returned `len(d[55:])` when I tested it on a router running newer RouterOS version, so that is where I got that value from.

Reply:
According to #24, RouterOS version 6.43 is giving an error before reaching this like, so don't know how to detect that.

Reply:
Nevermind, #24 seem to be an error given when the Winbox service has a "allowed from" accesslist.
Fixed in #25